.github/workflows/need-attention-label.yml

```diff
@@ -0,0 +1,22 @@
+name: Add need attention label
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  apply-label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v3
+        # the login of our bot called 'sam-cli-bot'
+        if: github.event.sender.login != 'sam-cli-bot'
```

Comment:
Didn't we say at some point that we would use another user for this?

Reply:
The new bot is called sam-cli-bot. I think you mentioned the old one, which is aws-sam-cli-bot

Reply:
The names are really close.
Shouldn't the new one be named: `aws-sam-stale-bot`?

Reply:
changed it to aws-sam-cli-stale-bot